src/pedalintandem.py

```diff
@@ -0,0 +1,114 @@
+import json
+from datetime import datetime
+from curl_cffi import requests
+from common.tz import IST
+from bs4 import BeautifulSoup
+
+BASE_URL = "https://www.pedalintandem.com"
+
+def _date(date_str):
+    return datetime.fromisoformat(date_str).astimezone(IST).isoformat()
+
+
+def fetch_events_links(session):
+	res = session.get(f"{BASE_URL}/experiences", impersonate="chrome")
+	soup = BeautifulSoup(res.text, 'html.parser')
+	event_divs = soup.find_all('div', class_ = 'single-experience')
+
+	# Fetch events from other pages.
+	load_more = soup.find('div', class_ = 'products-loadmore')
+	while load_more != None:
+		url = load_more.find('a').get('href')
+		
+		new_page = session.get(f"{BASE_URL}{url}")
+		soup = BeautifulSoup(new_page.text, 'html.parser')
```

Comment:
do not reuse soup here, causes confusion on the next few lookups on whether the content is on the first page or the others.

Reply:
Ok. Will change it